### PR TITLE
Create the method to clear collections of feature

### PR DESCRIPTION
**Pull request by duduribeiro**
Add a way to clear collections from a specific feature.

### CODE REVIEW

File: spec/rollout_spec.rb [hunk 1]
```diff
@@ -305,6 +305,21 @@
 
   end
 
+  describe '#clear_collections_from_feature' do
+    before do
+      @rollout.define_id_collection(:will_remove,     [200, 205])
+      @rollout.define_id_collection(:will_remove_too, [300, 305])
+      @rollout.add_collection_to_feature(:will_remove,    :my_feature)
```
Comment (GonchuB) [inline spec/rollout_spec.rb]: if u are gonna add extra spaces for this to be aligned, u are missing one space, if not, remove extra spaces :P

Comment (duduribeiro) [inline spec/rollout_spec.rb]: what do you guys prefer? :D

Comment (cgraio) [inline spec/rollout_spec.rb]: todo alineado queda más lindo :+1:

**Comment by duduribeiro** [comment]
fixed \o

**Comment by GonchuB** [comment]
Let me know when the pr on the ui is ready =)

**Comment by duduribeiro** [comment]
Only waiting this to be merged, so I can use the correct branch :smile:

Can I merge it? thanks

2015-09-29 17:07 GMT-03:00 Gonzalo Beviglia notifications@github.com:

**Pull request merged**